GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.cs

```diff
@@ -409,7 +409,7 @@ private void Clone(IHostedRepository repo)
 
             var module = new GitModule(targetDir);
 
-            if (addUpstreamRemoteAsCB.Text.Trim().Length > 0 && !string.IsNullOrEmpty(repo.ParentReadOnlyUrl))
+            if (addUpstreamRemoteAsCB.Text.Trim().Length > 0 && !GitExtensions.Strings.IsNullOrEmpty(repo.ParentReadOnlyUrl))
```

Comment:
```suggestion
            if (addUpstreamRemoteAsCB.Text.Trim().Length > 0 && !Strings.IsNullOrEmpty(repo.ParentReadOnlyUrl))
```

Reply:
The `Strings` utility method has a naming conflict with a `Strings` resource dictionary, hence the fully qualified name here and elsewhere. In future perhaps we can rename one of the files to avoid this.